tr55/makeMiniAppTable.py

```diff
@@ -55,8 +63,11 @@
             }
         }
 
-        model_out = (simulate_day(cells, precip)
-                     ['unmodified']['distribution'].values()[0])
-        writer.writerow((precip, land_use,
-                         soil_type, model_out['et'],
-                         model_out['inf'], model_out['runoff']))
+        model_out = simulate_day(cells, precip)['unmodified']
```

Comment:
The output of `simulate_day` is a dictionary with keys `modified` and `unmodified`, and within each of those is a tree-like structure similar to the sample output given here: https://github.com/WikiWatershed/tr-55 .  So after that line is expected, `model_out` is equal to something which has keys `et`, `inf`, and `runoff` holding those respective quantities.